todo_app/integration_test.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+import dotenv
+from dotenv import load_dotenv
+import os
+from os.path import join, dirname
+from dotenv import find_dotenv
+from todo_app.app import create_app
+from unittest.mock import patch
+import responses
+import requests
+import vcr
+
+@pytest.fixture
+def client():
+# Use our test integration config instead of the'real' version
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+# Create the new app.
+#test_app = app.create_app()
+    test_app = create_app()
+# Use the app to create a test_client that can be used in our tests.
+    with test_app.test_client() as client:
+        yield client
+
+
+@patch('requests.get')
+def test_index_page(mock_get_requests, client):
+# Replace call to requests.get(url) with our own function
+    mock_get_requests.side_effect = mock_get_lists
+    test_board_id = '6052828032dafa5707bf5reg'
+    url = f'https://api.trello.com/1/boards/{test_board_id}/lists'
+    response = client.get(url)
```

Comment:
Here you're hitting trello directly, rather than hitting your application to test that its working - so this test isn't actually testing our application at all